crates/epaint/src/shadow.rs

```diff
@@ -11,27 +11,34 @@ pub struct Shadow {
 
     /// Color of the opaque center of the shadow.
     pub color: Color32,
+
+    /// Y-offset for the top edge of the rectangle. Positive values make the shadow
+    /// smaller, negative values make the shadow larger. The bottom edge is unchanged.
+    pub top_offset: f32,
```

Comment:
This will leave users wanting to do the same for the other three sides. You could use `Margin` here instead of `f32`, allowing users to tweak all four sides. You could use `Margin::expand_rect` to easily tweak the resulting rectangle.

I think it makes more sense that large values means "expand", so that to offset the whole shadow down and to the right (make it more like a drop-shadow) you could use `Margin { left: -8., right: 8., top: -8., bottom: 8. }`, for instance